volumina/pixelpipeline/imagesources.py

```diff
@@ -689,30 +689,32 @@ def request( self, qrect, along_through=None ):
 from volumina.utility.segmentationEdgesItem import SegmentationEdgesItem, generate_path_items_for_labels
 class SegmentationEdgesItemSource(ImageSource):
     def __init__( self, layer, arraySource2D ):
-        from volumina.layer import SegmentationEdgesLayer
+        from volumina.layer import SegmentationEdgesLayer, LabelableSegmentationEdgesLayer
         assert isinstance(layer, SegmentationEdgesLayer)
         
         super( SegmentationEdgesItemSource, self ).__init__(layer.name)
         self._arraySource2D = arraySource2D
         self._arraySource2D.isDirty.connect(self.setDirty)
         self._layer = layer
+        self._is_clickable = isinstance(self._layer, LabelableSegmentationEdgesLayer)
```

Comment:
It's better to avoid `isinstance` checks, as they can be confusing because they provide implicit interface.
We can define it explicitly for example with property `Layer.has_clickable_item`
Also, this information can be omitted here, because we pass `self._layer` as an argument to `SegmentationEdgesItemRequest`